generator/shared.go

```diff
@@ -339,39 +339,47 @@ type GenOpts struct {
 	Copyright         string
 }
 
-// TargetPath returns the target path relative to the server package
+// TargetPath returns the target generation path relative to the server package
+// May be used in templates. Errors are not fatal: an empty string is returned instead
 func (g *GenOpts) TargetPath() string {
```

Comment:
Why are errors not fatal?

Reply:
Actually I didn't find where this method could be used in the code. The only use I could find was through templates: hence forcing empty string